zetaclient/orchestrator/orchestrator.go

```diff
@@ -57,40 +74,57 @@ type Orchestrator struct {
 	// last operator balance
 	lastOperatorBalance sdkmath.Int
 
-	// misc
+	// logger contains the loggers used by the orchestrator
 	logger Log
-	stop   chan struct{}
-	ts     *metrics.TelemetryServer
+
+	// dbPath is the path observer database
+	dbPath string
+
+	// ts is the telemetry server for metrics
+	ts *metrics.TelemetryServer
+
+	// mu protects fields from concurrent access
+	mu sync.Mutex
+
+	// stop channel and flag to avoid closing twice
+	stop    chan struct{}
+	stopped bool
```

Comment:
Maybe inactive instead of stopped?

Would also be great to have comment for the field

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/4f61c380bd8629ea044d7284eabf99c09b447131